tests/commands/edit_flags.py

```diff
@@ -0,0 +1,56 @@
+"""
+`edit-flags` command test module
+"""
+
+
+import pytest
+
+from tests.utils import (
+    ARCH,
+    GefUnitTestGeneric,
+    gdb_start_silent_cmd_last_line,
+    gdb_start_silent_cmd,
+)
+
+
+class EditFlagsCommand(GefUnitTestGeneric):
+    """`edit-flags` command test module"""
+
+    def setUp(self) -> None:
+        res = gdb_start_silent_cmd_last_line("edit-flags")
+        self.assertNoException(res)
+        flags = res[1:-1].split()
+        self.flag_name = "carry" # todo: update with other arches
```

Comment:
todo?

Reply:
the todo can be ignored for now (it was already in `runtests.py`)
currently that flag works for x86, x64, arm & aarch64 which are the archs I can test the most frequently. if/when we get a smoother way to test on the other arches, it can be updated